IDEAS/Buildings/Components/OuterWall.mo

```diff
@@ -88,7 +104,11 @@ protected
     redeclare package Medium = Medium,
     final table=coeffsCp,
     final azi=aziInt,
-    Cs=Cs,
+    Cs=if not Use_custom_Cs and sim.interZonalAirFlowType == IDEAS.BoundaryConditions.Types.InterZonalAirFlow.TwoPorts
+         then sim.Cs_coeff*Habs 
```

Comment:
`then sim.Cs_coeff*Habs `

Should become:

`then sim.Cs_coeff*(Habs^(2*sim.a))`